corehq/apps/couch_sql_migration/progress.py

```diff
@@ -52,3 +54,7 @@ def set_couch_sql_migration_complete(domain):
     # we get this for free
     set_tz_migration_complete(domain)
     _notify_dimagi_users_on_domain(domain)
+
+
+def total_couch_domains_remaining():
+    return DomainES().AND(filters.term('use_sql_backend', False)).count()
```

Comment:
I don't think you need the `.AND`, you can just go `DomainES().filter(filters.term('use_sql_backend', False)).count()`

Also, if you want to include `snapshot` domains (ones used for the exchange) then you can add `.remove_default_filters()`

Reply:
6b78aa8 I don't know too much about how HQES works, but is there a reason for that change for filter? It makes more sense semantically, but I guess I should try to look at the raw queries at some point

2513e9d remove_default_filters